### DISCUSSION TITLE
remove 80% limit on loan interest

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17999] [Feature requests] [answered]
I would like to remove the 80% loan interest cap, and allow annual loan interest of up to 144% when creating scenarios

https://github.com/OpenRCT2/OpenRCT2/pull/18000

**Comment by Gymnasiast** [comment]
Can you perhaps explain the reason behind it? Especially why you think 80% is too low, and why it should be 144% specifically.

**Reply by Gymnasiast** [reply]
Or perhaps we should ask @SpartanFrederic104 (who made a few hard scenarios) if this is something he thinks is useful.

**Comment by 733737** [comment]
I have a friend who has been waiting for this feature for some time. The idea behind it is similar to a pay day loan, RCT already has scenarios where a park is mismanaged or in disrepair, this could create a new financial hurdle for the player. I also think that this feature would be welcomed by the community... I chose 144% because when I tried 400% the compile failed and said unsigned char changes to 144 so I went with that and it compiled

**Reply by James103** [reply]
Could the max loan interest be raised to at least 250 or 255%? 400% is too high since loan interest (as a percentage) is stored as an 8-bit integer whose max value is 255, hence the warning and subsequent compilation failure.

**Reply by 733737** [reply]
if you want to make scenarios with 255% interest and Gymnasiast is ok with it being that high, I could easily change the code

**Reply by ocalhoun6** [reply]
It would be hard to balance if you were trying to make a scenario challenging this way.

For scenarios that start with more debt than cash, there's a large region where the interest rate is too low and is relatively easy to pay off the loan and get out of it, a large region where the interest rate is too high and the player gets caught in a debt spiral, and a knife's edge little sliver between the two where it significantly increases the challenge, but is still doable.

For scenarios that start with equal or less debt than cash, the obvious choice for a high interest loan is to just pay off the loan immediately and keep the loan balance at zero. Maybe you could add this to a scenario as a way of discouraging loan use ... but you could also do the same by setting a low loan limit or not allowing loans at all.

So, to me, it seems that it might _sometimes_ be helpful in making challenging scenarios ... but only marginally helpful.

**Comment by 733737** [comment, marked as answer]
this was implemented a while ago
